frontend/src/styles/_aepp-list.scss

```diff
@@ -1,13 +1,13 @@
-.app-list {
+.aepp-list {
   margin-top: 3rem;
   padding: 0;
   padding-bottom: 5rem;
 }
 
-.applist-row {
+.aepp-list-item {
   background-color: $item-color;
   display: flex;
-  box-shadow: 0 0.25rem 0.25rem rgba($text-color, 0.1);
+  box-shadow: 0 0.25rem 0.25rem $list-item-shadow-color;
```

Comment:
Do not use $text-color variable as color for shadow color.
Create new variable for #100c08 color then assign this variable to $text-color and $list-item-shadow-color
e.g. $black: #000;
$text-color: $black; (this is just example with black color)